lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -422,24 +422,44 @@ def compile_methods(name, constant)
 
         sig { params(module_name: String, mod: Module, for_visibility: T::Array[Symbol]).returns(String) }
```

Comment:
Do we want to keep `for_visibility` as an argument for the future? It's not configured anymore.

Reply:
Yes, I thought it might come in handy, so didn't remove it.